tests/ci/azureml_tests/test_groups.py

```diff
@@ -90,8 +90,8 @@
         #
         "tests/smoke/examples/test_notebooks_gpu.py::test_wide_deep_smoke",  # 122.71s
         #
-        "tests/smoke/examples/test_notebooks_gpu.py::test_fastai_smoke",  # 33.22s
-        "tests/functional/examples/test_notebooks_gpu.py::test_fastai_functional",  # 667.88s
+        "tests/smoke/examples/test_notebooks_gpu.py::test_embdotbias_smoke",  # 30.99s
+        "tests/functional/examples/test_notebooks_gpu.py::test_embdotbias_functional",  # 29.10s
```

Comment:
```suggestion
        "tests/unit/recommenders/models/test_embdotbias_model.py", 
        "tests/unit/recommenders/models/test_embdotbias_utils.py",
        "tests/smoke/examples/test_notebooks_gpu.py::test_embdotbias_smoke",  # 30.99s
        "tests/functional/examples/test_notebooks_gpu.py::test_embdotbias_functional",  # 29.10s
```

Adding new tests to the groups so they are executed in the AzureML VMs